### DISCUSSION TITLE
Clarification on the use of JSONResponse (json.dumps)

### DISCUSSION THREAD

**Discussion by ni-aas** [encode/starlette#2345] [Q&A] [answered]
I see that the `JSONResponse` class in Starlette have a `render` method to handle the conversion of data to JSON format. 

But, it's necessary to separately call json.dumps when creating a JSONResponse like this: `JSONResponse(json.dumps(message))`? 
If not I get "not valid JSON" on the client side.

Thanks for your time and help.

**Comment by Kludex** [comment, marked as answer]
Which client are you using?

I need an MRE.
